SparkPost/2web2ui
# AC-1978: Global tracking for all forms in recipientValidation, register and messageEvents

AC-1978: Global tracking for all forms in recipientValidation, register and messageEvents

### What Changed

- Added hasError key with form submission data (Jon's feedback)
- Allowlisted forms of recipientValidation, register and messageEvents
- Added cypress tests for the above forms
- Added unit tests for redux forms 

### How To Test

- Check for form submissions on recipientValidation and check if all the sensitive fields are blocklisted
- Check for form submissions on register page.
    > Invite a user and click on the link in the invitation email
- Check for form submissions on Events page 

### To Do
